.github/workflows/creating_pr.yml

```diff
@@ -0,0 +1,43 @@
+name: Create Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to release'
+        required: true
+        default: '1.0.0'
```

Comment:
it sounds like we may have `1.0.0`  often, can we remove the default?

Reply:
yes sure !